### ISSUE TITLE
Post game not triggering?

### ISSUE THREAD

**Issue by Midgettt** [Synthlight/Nexomon-Extinction-Save-Editor#5]
I used the save editor to edit 2 nexomon in my storage which are now part of my main party but when i beat the game and reload my save i'm sent back to the spot before the airship fight where the previous nexomon protag gives me the option to leave. But then i watch some youtube on a post game only to see the youtuber "HeroVolsty" beat the game and was sent back to the guild hall like theres a post game.
did the save editor do this? or is it a different issue. i hope we can sort this out, your editor saved me uncountable amounts of hours looking for extinct nexomon.

**Comment by Synthlight** [comment]
You'd need to try with the save before the edit to really confirm if that's the case.
But the editor doesn't touch story/progression stuff at all. It's possible the format changed or something, but again, this would need the before & after save to determine what, if anything, went wrong.

The editor only loads the first part of the save:
![image](https://user-images.githubusercontent.com/1075166/121780299-b09ec100-cb6d-11eb-914f-46ea9f0f039d.png)
Everything after this (all the story shit) is preserved without any alterations.

In short, if you want this investigated, I'd need:
- The save before & after editing.
- What platform is this happening on. (As the save formats differ a bit in that case.)

But my gut says it's just a game bug? Or maybe you didn't meet some condition you needed to?
Or maybe you edited in some nexomon that was tied to the story and the quest didn't know how to handle that situation and some flag wasn't set?
Without the saves, I really can't say.

**Comment by Midgettt** [comment]
I have the unaltered and alter saves
The altered has the game completed
The unaltered save is a while back i think before being sent to prison

I play on Pc

**Comment by Synthlight** [comment]
Drop the unaltered save here, and either a copy of it that has the same edits you made, or tell me what edits you made (or both, but knowing the changes would be more helpful).
I'll go comparing the two and see if it did anything it shouldn't have, if any data was lost/corrupted, etc.

**Comment by Midgettt** [comment]
unmodded save file:

**Comment by Midgettt** [comment]
modded save file:

**Comment by Midgettt** [comment]
what i did with the modded save file was i edited 2 nexomon in my storage
to be Florozard and Glunch. I forgot what the original nexomon were before
they were edited.

an extra detail would be that both glunch and florozard were edited to be
level 15 and in my storage so they could level up and naturally get their
move pools and stats

**Comment by Synthlight** [comment]
Emailing these is appears not to have attached any files.

**Comment by Midgettt** [comment]
That's weird. They show up on my end.

**Comment by Synthlight** [comment]
![image](https://user-images.githubusercontent.com/1075166/121782974-480a1100-cb7a-11eb-8a13-19085441cfe7.png)

**Comment by Midgettt** [comment]
github doesnt support the file type that the save data's are. would you like me to send a google drive link instead?

**Comment by Synthlight** [comment]
Or just zip them and upload that. Either works. (Though the latter will keep the saves preserved as long as the issue lives, whereas external links may die.)

**Comment by Midgettt** [comment]
I’ve made a drive folder which has both the modded and un-modded save files

https://drive.google.com/drive/folders/1b9A-mBOyhpgeAJJrOzqc0ngl-qpg1DAQ

If it makes it easier or provides a different result i can also zip the
files and send them to you but that will have to wait until tomorrow as it
is currently 2:54am where i live and i need to go tf to sleep.

Thanks for all the help you’ve given me so far i hope this’ll work out.

**Comment by Midgettt** [comment]
Update: after tweaking the save file i was able to get the post game to
trigger after defeating Vados a second time.
Thanks for your help.

**Comment by Synthlight** [comment]
What exactly did you tweak? Was it something the editor did or something you did?
